tests/gss/check-context.in

```diff
@@ -146,9 +146,15 @@ ${context} --client-name=user1@${R} --client-password=u2 \
 ${klist} && { eval "$testfailed"; }
 
 echo "Getting client initial tickets" ; > messages.log
-${kinit} --password-file=${objdir}/foopassword --forwardable user1@${R} || \
+${kinit} --no-pk-anon-fast-armor --password-file=${objdir}/foopassword --forwardable user1@${R} || \
```

Comment:
`--no-pk-anon-fast-armor` is needed to avoid FAST for this test.

Reply:
Note that for this test we're setting `[kdc]` `max-kdc-datagram-reply-length` to 1200.